front/app/containers/IdeasShow/index.tsx

```diff
@@ -61,76 +58,45 @@ export const IdeasShow = ({
   ideaId,
   setRef,
 }: Props) => {
-  const { data: ideaImages } = useIdeaImages(ideaId);
-  const { data: idea } = useIdeaById(ideaId);
-  const { data: project } = useProjectById(projectId);
+  const { data: ideaImages, isLoading: isLoadingIdeaImages } =
+    useIdeaImages(ideaId);
+  const { data: idea, isLoading: isLoadingIdea } = useIdeaById(ideaId);
+  const { data: project, isLoading: isLoadingProject } =
+    useProjectById(projectId);
+  const localize = useLocalize();
+  const [translateButtonIsClicked, setTranslateButtonIsClicked] =
+    useState(false);
   const postOfficialFeedbackPermission = usePermission({
-    item: !isNilOrError(project) ? project.data : null,
+    item: project?.data || null,
     action: 'moderate',
   });
-
-  const isLoaded = !!(idea && ideaImages && project);
+  const { data: phases } = usePhases(project?.data.id);
 
   const handleContainerRef = (element: HTMLDivElement) => {
     setRef?.(element);
   };
 
-  return (
-    <Container
-      projectId={projectId}
-      isLoaded={isLoaded}
-      className={className}
-      handleContainerRef={handleContainerRef}
-    >
-      {isLoaded && (
-        <Content
-          postOfficialFeedbackPermission={postOfficialFeedbackPermission}
-          idea={idea}
-          project={project.data}
-          ideaImages={ideaImages}
-          compact={compact}
-        />
-      )}
-    </Container>
-  );
-};
-
-interface ContentProps {
-  postOfficialFeedbackPermission: boolean;
-  idea: IIdea;
-  project: IProjectData;
-  ideaImages: IIdeaImages;
-  compact: boolean;
-}
-
-const Content = ({
-  postOfficialFeedbackPermission,
-  idea,
-  project,
-  ideaImages,
-  compact,
-}: ContentProps) => {
-  const { data: phases } = usePhases(project.id);
-  const localize = useLocalize();
-  const [translateButtonIsClicked, setTranslateButtonIsClicked] =
-    useState(false);
+  if (isLoadingIdea || isLoadingIdeaImages || isLoadingProject) {
+    return <FullPageSpinner />;
+  }
 
-  const authorId = idea.data.relationships?.author?.data?.id || null;
-  const statusId = idea.data.relationships?.idea_status?.data?.id;
-  const ideaImageLarge =
-    ideaImages?.data[0]?.attributes?.versions?.large || null;
-  const ideaId = idea.data.id;
-  const ideaBody = localize(idea.data.attributes?.body_multiloc);
+  if (!idea || !project) {
+    return null;
+  }
 
+  const authorId = idea.data.relationships.author?.data?.id || null;
+  const statusId = idea.data.relationships.idea_status.data?.id;
+  const ideaImageLarge = ideaImages?.data[0]?.attributes?.versions?.large;
   const participationContext = getCurrentPhase(phases?.data);
-
-  const inputTerm = getInputTerm(phases?.data);
-
   const wasImported = !!idea.data.relationships.idea_import?.data;
 
   return (
-    <>
```

Comment:
The Content component was moved directly inside Container, making it look like I've added a new wrapper.